Automattic/woocommerce-services
# Update babel to version 6

To test:
- checkout this branch
- `rm -rf node_modules`
- `npm cache clean`
- `npm install`
- `npm start`
- verify that the settings page loads and works properly and no errors display anywhere
- `npm test` -- all tests should pass
- `npm dist` -- make sure you can compile the "built" app

Future PRs will have other improvements now that we have this in place

cc @jeffstieler @allendav @nabsul for review
cc @coderkevin @aduth  for general interest
